sources/window/error_source_deprecated.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package window
+
+import (
+	"time"
+
+	"github.com/palantir/witchcraft-go-health/conjure/witchcraft/api/health"
+)
+
+// MustNewUnhealthyIfAtLeastOneErrorSource returns the result of calling NewUnhealthyIfAtLeastOneErrorSource, but panics if it returns an error.
```

Comment:
It is technically correct because it `returns the result of calling New`. It doesn't `calls New`.
I think we can leave this as it's just the legacy GoDoc for a deprecated endpoint.

Reply:
I didn't want to forward the call because I wanted it to give the example of how the health check should be created using the options.